src/Tribe/Views/V2/View.php

```diff
@@ -850,6 +853,9 @@ public function prev_url( $canonical = false, array $passthru_vars = [] ) {
 				$url = add_query_arg( $query_args, $url );
 			}
 
+			// Remove the inverse of the page key we are using.
+			$url = remove_query_arg( 'page' === $this->page_key ? 'paged' : 'page', $url );
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :no_entry_sign: Usage of Yoda conditions is not allowed; switch the expression order
* :warning: Line exceeds 80 characters; contains 91 characters

<sub>hash: 3c955d5123c3a49b02cf78fd70e2e58b</sub>